core/web/resolver/query.go

```diff
@@ -440,6 +440,25 @@ func (r *Resolver) Config(ctx context.Context) (*ConfigPayloadResolver, error) {
 	return NewConfigPayload(printer.EnvPrinter), nil
 }
 
+// ConfigV2 retrieves the Chainlink node's configuration (V2 mode)
+func (r *Resolver) ConfigV2(ctx context.Context) (*ConfigV2PayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	cfg := r.App.GetConfig()
+	if cfg.RawUser() == "env" {
```

Comment:
IMHO it would be more clear to omit these methods from the interface rather than encode special strings inside of their values.
We can just check for a new config interface method, e.g.: `newCfg, ok := cfg.(interface{ func TOMLConfig() (user, effective string) })`